scripts/provisioning-tests

```diff
@@ -1,28 +1,80 @@
 #!/bin/bash
-set -e
+set -e -x
+
+cleanup()
+{
+    EXIT=$?
+    set +ex
+    echo Stopping rancher server
+    kill $RANCHER_RUN_PID
+    wait $RANCHER_RUN_PID
+    if [ $PID != -1 ]; then
+      kill $PID
+      wait $PID
+    fi
+    return $EXIT
+}
 
 cd $(dirname $0)/..
 
+SOME_K8S_VERSION=v1.21.5+rke2r2
```

Comment:
Can we put a comment here indicating that this should be removed when a proper version of RKE2 is added to KDM to address the the problem?